po/common/en.po

```diff
@@ -1709,6 +1709,10 @@ msgid "The table lists by default nutriments that are often specified. Leave the
 msgstr "The table lists by default nutriments that are often specified. Leave the field blank if it's not on the label.<br/>You can add extra nutriments (vitamins, minerals, cholesterol etc.)\n"
 "by typing the first letters of their name in the last row of the table."
 
+msgctxt "nutrition_data_table_asterisk"
+msgid "* : essential nutiments to calculate nutri-score."
```

Comment:
```suggestion
msgid "essential nutrients to calculate the Nutri-Score."
```
It's better to keep the * out of the string. You can put it in the template instead, like this:

A few comments:

`*[% sep %]:[% lang('nutrition_data_table_asterisk') %]`

The [% sep %] is used to put a space in some languages that need a space before the : character (e.g. French but not English).

We decided to use the word "nutrient" instead of the lesser used "nutriment".
"Nutri-Score" is a trademark, it must be capitalized that way.

Reply:
Okay...fixing it!


> It's better to keep the * out of the string. You can put it in the template instead, like this:
> 
> A few comments:
> 
> `*[% sep %]:[% lang('nutrition_data_table_asterisk') %]`
> 
> The [% sep %] is used to put a space in some languages that need a space before the : character (e.g. French but not English).
> 
> We decided to use the word "nutrient" instead of the lesser used "nutriment". "Nutri-Score" is a trademark, it must be capitalized that way.

Okay...fixing it 👍